dashboard/app/models/course.rb

```diff
@@ -212,6 +237,34 @@ def summarize
     }
   end
 
+  # If a user has no experiments enabled, return the default set of scripts.
+  # If a user has an experiment enabled corresponding to an alternate script in
+  # this course, use the alternate script in place of the default script with
+  # the same position.
+  # @param user [User]
+  # @return [Array<Script>]
+  def scripts_for_user(user)
+    return default_scripts unless user
```

Comment:
Should we also early exit in the case where we have a user, but the user has no experiments?